README.md

```diff
@@ -17,6 +17,7 @@ Supported compression formats:
 * stored (i.e. none)
 * deflate
 * bzip2
+* zstd (in progress...)
```

Comment:
Remove "in progress..." ? Since with merging this PR we assume that `zstd` algorithm is supported